### PR TITLE
Added unit test: surface.get_pitch()

### PR DESCRIPTION
**Pull request by anguye13**
[#1808 ](https://github.com/pygame/pygame/issues/1808)

### CODE REVIEW

File: test/surface_test.py [hunk 1]
```diff
@@ -1388,7 +1388,29 @@ def todo_test_get_pitch(self):
         #
         # This value is not needed for normal Pygame usage.
 
-        self.fail()
+        # Test get_pitch() on several surfaces of varying size
+        sizes = ((2, 2), (7, 33), (33, 7), (2, 734), (734, 2), (734, 734))
+        for width, height in sizes:
```
Comment (MyreMylar) [inline test/surface_test.py:1393]: My main thought here is - bit depths. It should be possible to add them into your tuples and then the creation of Surfaces and different bit depths should affect the values returned via `get_pitch()`, though if it works right it won't affect your tests as they start with `get_buffer()` which should also be affected in the same way.

**Comment by anguye13** [comment]
I made the changes :)

**Review by MyreMylar — APPROVED** [review]
Excellent! Approved 👍 🎉

**Review by illume — APPROVED** [review]
🎉 thank you.

**Pull request merged**